pyproject.toml

```diff
@@ -7,8 +7,7 @@ requires = [
   # keep in sync with travis.yml "minimal" specs (Cython and numpy for py36)
   "Cython>=0.26.1",
   "numpy==1.13.3; python_version=='3.6' and platform_system!='AIX'",
-  "numpy==1.18.1; python_version=='3.7' and platform_system!='AIX'",
-  "numpy==1.18.4; python_version=='3.8' and platform_system!='AIX'",
+  "numpy>=1.18.4; python_version>='3.7' and platform_system!='AIX'",
```

Comment:
Looks good to me, but together with #2917 , why don't we add support for Python 3.9 in the CI altogether?

Reply:
Of course that's the next step, but I'd rather have the bug fix go in asap while the testing can wait imo :)

Reply:
Fair enough!